kratos/utilities/model_part_graph_utilities.cpp

```diff
@@ -0,0 +1,256 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//
+
+// System includes
+#include <queue>
+
+// External includes
+
+
+// Project includes
+#include "containers/sparse_contiguous_row_graph.h"
+#include "utilities/parallel_utilities.h"
+#include "utilities/model_part_graph_utilities.h"
+
+namespace Kratos
+{
+    
+    SparseContiguousRowGraph<> ModelPartGraphUtilities::ComputeGraph(const ModelPart& rModelPart){
```

Comment:
I will be perfect if you move 4 spaces the code...

Reply:
formatted by astyle